roles/reproducer/tasks/configure_controller.yml

```diff
@@ -271,10 +271,16 @@
 
     - name: Inject kubeadmin-password if exists
       when:
-        - _devscripts_kubeadm.content is defined
+        - _devscripts_kubeadm.content is defined or
+          _crc_kubeconfig.content is defined
       ansible.builtin.copy:
         dest: "/home/zuul/.kube/kubeadmin-password"
-        content: "{{ _devscripts_kubeadm.content | b64decode }}"
+        content: >-
+          {{
+            (_devscripts_kubeadm.content is defined) |
+            ternary(devscripts_kubeadm.content, _crc_kubeadm.content) |
```

Comment:
typo devscripts_kubeadm -> _devscripts_kubeadm

Seeing failures:- The task includes an option with an undefined variable. The error was: 'devscripts_kubeadm' is undefined. 'devscripts_kubeadm' is undefined